src/server/context.ts

```diff
@@ -157,6 +160,40 @@ export class ServerContext {
       );
     }
 
+    // Restore snapshot if available
+    let snapshot: BuildSnapshot | null = null;
+    // Load from snapshot if not explicitly requested not to
+    const loadFromSnapshot = opts.loadSnapshot ?? true;
+    if (loadFromSnapshot) {
```

Comment:
Is this actually backwards compatible?

Reply:
Not sure what you mean. This is the first time we're exposing the snapshot.